dist/pym.js

```diff
@@ -482,7 +487,11 @@
             */
 
             // Get the child's height.
-            var height = document.getElementsByTagName('body')[0].offsetHeight.toString();
+            var height = "0";
+            var body = document.getElementsByTagName('body')[0];
+            if(body) {
+                height = body.offsetHeight.toString();
+            }
```

Comment:
In firefox, when multiple pym windows are on the same page, I'm seeing the 'body' tag come up undefined sometimes. It's inconsistant and highly aggrivating.